lib/matplotlib/colorbar.py

```diff
@@ -854,10 +857,22 @@ def get_ticks(self, minor=False):
             return self._long_axis().get_majorticklocs()
 
     @_api.delete_parameter("3.5", "update_ticks")
-    def set_ticklabels(self, ticklabels, update_ticks=True):
+    def set_ticklabels(self, ticklabels, *, minor=False, update_ticks=True,
+                       **kwargs):
```

Comment:
`update_ticks` is still positional.
```suggestion
    def set_ticklabels(self, ticklabels, update_ticks=True, *, minor=False,
                       **kwargs):
```